tests/test_convert.py

```diff
@@ -6,6 +6,8 @@ def test_convert():
     pifs = list(convert(["./test_files/template_example.csv"]))
     assert pifs[0].names[0] == 'P20 Tool steel'
     assert pifs[0].sub_systems[0].names[0] == 'Martensite'
+    assert pifs[0].classifications[0].name == 'Test Classification'
```

Comment:
how come this is only `Test Classification` and not `CLASSIFICATION: Test Classification`? where does the `CLASSIFICATION:` get filtered out?

Reply:
@yoolheekim this is testing the PIF that is created and the `CLASSIFICATION:` keyword is just part of the CSV. When it gets converted to a PIF the keyword is dropped and only the name specified is stored in the name field. This is the same behavior as for `IDENTIFIER:` etc.